docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable and latest releases
+
+The following release series include:
+
+- [Stable](../versioning/#stable-releases): Supported for production deployments.
+- [Latest](../versioning/#latest-releases): Supported for development and testing only.
+
+[Release versioning](../versioning) gives details on the stable and latest release series numbering convention that started with `2.2`.
+
+[Planned releases](#planned-releases), in a section below, includes timelines for upcoming stable and latest release series.
+
+| Release series                    | Released     | Maintenance support ends | EOL          | Status                              |
+| :-------------------------------- | :----------- | :----------------------- | :----------- | :----------------------------------- |
+| [2.3](../whats-new)               | Sep 08, 2020 | ---                      | ---          | Latest (development and testing)
+| [2.2](../earlier-releases/v2.2.0) | Jul 15, 2020 | Jul 15, 2021             | Jan 15, 2022 | Stable (recommended for production)
+| [2.1](../earlier-releases/v2.1.0) | Feb 25, 2020 | Feb 25, 2021             | Aug 08, 2021 | Stable
+| [2.0](../earlier-releases/v2.0.0) | Sep 17, 2019 | Sep 17, 2020             | Mar 03, 2021 | Stable
+| [1.3](../earlier-releases/v1.3.0) | Jul 15, 2019 | Jul 15, 2020             | Jan 15, 2021 | Stable
+
+**Note:** The stable release series include `2.1`, `2.0`, and `1.3`, released prior to the new [release versioning](../) convention.
+
+## Planned releases
```

Comment:
@stevebang Minor changes on this section
1. Can we reword it to "Upcoming releases"
2. Can we flip the order of version and planned release columns and sort it by the release date?

Reply:
done